docs/pages/integrations/index.md

```diff
@@ -13,7 +13,7 @@ abstract: Integrations with other platforms and services.
 
 
 {% assign sorted_index = site.pages | sort: "name" | alphabetical %}
-### All Integrations by Name (forloop.length)
+<!-- ### All Integrations by Name (forloop.length) -->
```

Comment:
Thanks, however, it would be nice to include a heading with total integration count. Please refactor.

Reply:
Does this code look good to you? Should I refactor code using this?
```
{{% assign sorted_index = site.pages | sort: "name" | alphabetical %}
{% assign integration_count = 0 %}

<!-- All Integrations by Name ({{ site.integrations.size }}) -->
<!-- Count the number of integrations -->
{% for item in sorted_index %}
  {% if item.type == "installation" and item.category == "integrations" and item.list == "include" and item.language == "en" %}
    {% assign integration_count = integration_count | plus: 1 %}
  {% endif %}
{% endfor %}

<!-- Display the total integration count with the message -->
<h2>All Integrations by Name ({{ integration_count }})</h2>
```
```

Reply:
@RipulHandoo, no, you should use built-ins from liquid templating - https://shopify.github.io/liquid/

Reply:
Got it. I'll update PR right away.